diesel/src/query_builder/debug.rs

```diff
@@ -2,7 +2,7 @@ use backend::Debug;
 use super::{QueryBuilder, BuildQueryResult};
 
 #[doc(hidden)]
-#[derive(Debug)]
+#[derive(Debug, Default)]
```

Comment:
I wish `Default` had `fn new() -> Self` as an alias for `Default`. Since we can derive this, and the derived fn will be identical to our explicit `new` function. Since this struct is purely internal, I wonder if it's worth just deleting our `new` function entirely and changing the call site to use `default`. It reads slightly worse, but if we're adding the derive we may as well have less code as a result.